dashboard/src/components/TestDetails/TestDetails.tsx

```diff
@@ -138,6 +142,76 @@ const TestDetailsSections = ({
     test.tree_name,
   ]);
 
+  const endTimestampInSeconds = dateObjectToTimestampInSeconds(
+    roundToNearestMinutes(new Date(), {
+      nearestTo: 30,
+    }),
+  );
+  const startTimestampInSeconds =
+    endTimestampInSeconds - daysToSeconds(REDUCED_TIME_SEARCH);
+
+  const hardwareDetailsLink = useMemo(() => {
+    if (hardware === formatMessage({ id: 'global.unknown' })) {
+      return <span>{hardware}</span>;
+    }
+
+    return (
+      <MemoizedLinkItem
+        to="/hardware/$hardwareId"
+        params={{ hardwareId: hardware }}
+        state={s => s}
+        search={{
+          origin: searchParams.origin,
+          startTimestampInSeconds: startTimestampInSeconds,
+          endTimestampInSeconds: endTimestampInSeconds,
+        }}
+      >
+        {hardware}
+        <LinkIcon className="text-blue text-xl" />
+      </MemoizedLinkItem>
+    );
+  }, [
+    searchParams,
+    hardware,
+    formatMessage,
+    startTimestampInSeconds,
+    endTimestampInSeconds,
+  ]);
+
+  const compatiblesLink = useMemo(() => {
+    if (!test.environment_compatible) {
+      return <span>{valueOrEmpty(test.environment_compatible, '-')}</span>;
+    }
+
+    return (
+      <>
+        {test.environment_compatible.map(
+          (compatible: string): JSX.Element => (
```

Comment:
IMO it would be better to leave them in a single line
```suggestion
    return (
      <div className="flex flex-wrap gap2">
        {test.environment_compatible.map(
          (compatible: string): JSX.Element => (
```